Dockerfile

```diff
@@ -0,0 +1,43 @@
+#  Copyright (C) 2021 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+FROM usdotfhwastoldev/carma-base:noetic-develop as base_image
```

Comment:
Change this to foxy-develop

I will push an image tonight

Reply:
For local testing you can tag your current noetic-develop as foxy-develop with the docker tag command.